.doc_gen/metadata/redshift_metadata.yaml

```diff
@@ -65,6 +119,24 @@ redshift_ModifyCluster:
   synopsis: modify an &RS; cluster.
   category:
   languages:
+    Kotlin:
+      versions:
+        - sdk_version: 1
+          github: kotlin/services/redshift
+          sdkguide:
+          excerpts:
+            - description: Create the cluster.
+              snippet_tags:
+                - redshift.kotlin.create_cluster.main
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/redshift
+          sdkguide:
+          excerpts:
+            - description: Modify a cluster.
+              snippet_tags:
+                - redshift.java2.create_cluster.main
```

Comment:
(optional) To my eye, I'd rather have these just contain the modifyCluster function, rather than start with the createCluster function.
And, for that matter, I'd take modifyCluster out of the createCluster snippets.
But it's up to you!

Reply:
It was tagged as 1 example. I created 2 different snippets